### ISSUE TITLE
Parse Hart CVRs for counties with multiple tabulators (unblock OC)

### ISSUE THREAD

**Issue by benadida** [votingworks/arlo#1446]

**Comment by ginvdr** [comment]
Another thing to consider when we do this is how a county might add a placeholder batch...  and probably discuss what that looks like scope wise!

**Comment by arsalansufi** [comment]
A related case for us to cover: When a jurisdiction uses different modes while scanning (we recently learned of a special audit mode). Like the multiple tabulators situation, this situation can also result in duplicate batch names. Unfortunately, unlike the multiple tabulators case, where CVRs can be exported by tabulator, CVRs can't be exported by mode. Scanned ballot information CSVs, however, can.

See https://votingworks.slack.com/archives/C01DUGG1D8E/p1660149845606469 for an example of this situation

**Comment by arsalansufi** [comment]
Tracking the file size concerns in a separate issue: https://github.com/votingworks/arlo/issues/1674
